src/control/Control.cpp

```diff
@@ -1385,10 +1147,9 @@ void Control::deletePage()
 	this->doc->unlock();
 
 	updateDeletePageButton();
-	this->undoRedo->addUndoAction(new InsertDeletePageUndoAction(page, pNr, false));
+	this->undoRedo->addUndoAction(mem::make_unique<InsertDeletePageUndoAction>(page, pNr, false));
```

Comment:
Can you explain why you are using make_unique?

Reply:
- It's efficient (perfect forwarding)
- It's clear
- It's short
- It's correct
- I don't repeat myself 
- I don't need to use new explicitly.
- Everyone knows what it does.

Reply:
It also have more benefits, a reviewer, does not care, if a 'new' Object is deleted, with that he knows "ah that's a unique_ptr, perfect, I don't need to check, whether this Object is deleted or not".
Also we have a clear ownership transfer. The code is correct by design :) .

Reply:
Ok, I didn't see the change of the List:
>std::deque<UndoActionPtr> undoList;

Then it's working.